tools/entry-node/docker-compose.yml

```diff
@@ -18,8 +18,9 @@ services:
       --autopeering.seed=${SEED}
       --autopeering.entryNodes=
       --analysis.client.serverAddress=
```

Comment:
What about this, shouldn't we disable the analysis.client plugin for the entrynode?

Reply:
Disabled analysis-client